src/components/Label/Label.scss

```diff
@@ -47,13 +47,28 @@ $hover-opacity: 0.7;
         font-size: var(--g-text-body-1-font-size);
         text-align: center;
         white-space: nowrap;
+        overflow: hidden;
+    }
+
+    &__content,
+    &__key {
         text-overflow: ellipsis;
         overflow: hidden;
     }
 
+    &__content,
+    &__key,
+    &__separator,
+    &__value {
+        vertical-align: top;
+
+        display: inline-block;
+    }
+
```

Comment:
It does nothing, all these elements are flex's children. Can be removed without any difference

Reply:
You're right, I guess it came from my first attempts to fix this the other way so I assumed it did something important